index.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+
+var stdio = require('stdio');
+var chalk  = require('chalk');
+var CLI    = require('clui');
+var figlet = require('figlet');
+var fs = require("fs");
+var fs_copy = require("fs-extra")
+var Spinner = CLI.Spinner;
+var path = require('path');
+var findRemoveSync = require('find-remove')
+
+
+
+console.log(
+    chalk.blue(
+        figlet.textSync('Report-Mine', { horizontalLayout: 'full' })
+    ))
+
+var ops = stdio.getopt({
+    'json': {key: 'results.json', args: 0
+        , description: 'Path to the json file'},
+});
+
+var mv = require('mv');
+var path = require('path');
+global = path.resolve(__dirname);
+
+var status = new Spinner('Generating Reports, please wait...');
+status.start();
+console.log(global)
+fs_copy.copy(global , process.argv[2]+"/report-mine" , function (err) {
+    if (err) {
+        status.stop();
```

Comment:
I mean , it will give a better idea to the user, what is going on wrong, for example, the error told me i have to give a dir and not the json file I generate